### ISSUE TITLE
Node.js 10 LTS

### ISSUE THREAD

**Issue by dshaw** [dshaw/alwaysforward#19]
## Title

Always Forward with dshaw: Node.js 10 LTS

## Topic

TBD

## Participant(s)

* Dan Shaw - @dshaw

CANDIDATES:
* 

## Schedule

TBD

## Agenda

No agenda

## Transcript

Not available

## Link(s)

* 

## Meta

#dshawaf #AlwaysForward #nodejs

**Comment by mcollina** [comment]
I’m in! Are we doing it in person in Vancouver?

**Comment by mhdawson** [comment]
Would be happy to participate.

**Comment by joyeecheung** [comment]
I will be landing in Vancouver that morning so unfortunately cannot participate :(

**Comment by mhdawson** [comment]
Just realized I'm also flying in on Oct 9 and will be in the air at 10 PT.  Could do later in the day, say 2PM EST?

**Comment by tzmanics** [comment]
@dshaw ping 👆

**Comment by dshaw** [comment]
I'm flying in after the show. We might have to rethink this scheduling a bit.

When does everyone get in? @mcollina @joyeecheung @mhdawson

**Comment by dshaw** [comment]
I land at 6pm. Could record at 8pm PT. 😬

**Comment by dshaw** [comment]
Dinner and a show?

**Comment by mcollina** [comment]
I land on Monday.

There is an event I’m going to in the evening of the 9th.
Il giorno mar 11 set 2018 alle 19:00 Dan Shaw <notifications@github.com> ha
scritto:

**Comment by mhdawson** [comment]
I could do 8PM

**Comment by joyeecheung** [comment]
I land around 10am on 9th, and will be flying the night before.

**Comment by tzmanics** [comment]
Another alternative would be to have this the week after Node Interactive (October 16th (Tues) 10a PT) to talk about the release and also share some stories about Collaborator Summit/Node Interactive. 

Would that work for everyone?

**Comment by joyeecheung** [comment]
10am PT on 16th works for me

**Comment by mhdawson** [comment]
10am PT on 16th works for me

**Comment by mcollina** [comment]
It does not work for me :/
Il giorno mar 11 set 2018 alle 21:45 Michael Dawson <
notifications@github.com> ha scritto:

**Comment by tzmanics** [comment]
Hey @Trott any chance you'd be free to join us 10am PT on 16th works for me?

**Comment by dshaw** [comment]
@mcollina I'm sorry to hear that you can't that time work. I totally understand. I hope we will be able to have you on again soon.

**Comment by Trott** [comment]
> Hey @Trott any chance you'd be free to join us 10am PT on 16th works for me?

I can probably make that work. I'm not sure I have much to add to a 10.0 LTS conversation that won't already be better covered by @joyeecheung and @mhdawson. If you're hoping I'll show up and be a Flavor Flav-style hype person to Joyee's more meaningful Chuck D flow, I can probably wear a huge clock.

Most of the recent releases of 10.x have been done by @targos. You might want to ask him. (Feel free to bump me for him if you can only have three people.) He probably has a better handle than most on what will and won't be important to highlight in 10.x LTS.

**Comment by joyeecheung** [comment]
Yes, I personally think @targos would be a better candidate than me (I still have trouble telling whether a PR should go into v10 or not quite often)

**Comment by targos** [comment]
Sorry, what is this?

**Comment by tzmanics** [comment]
Hi @targos sorry for the confusion, this is for a podcast with @dshaw covering the Node 10 LTS release basically talking about the major platform and language changes in the release. We would love for you to join the panel 10am PT on 10/16. Would you be interested?

Thanks for the recommendation @trott !

**Comment by targos** [comment]
I cannot make it on 10/16.
